telluric/georaster.py

```diff
@@ -1627,6 +1623,36 @@ def get_tile(self, x_tile, y_tile, zoom,
         affine = affine * Affine.scale(ratio, ratio)
         return self.get_window(window, bands=bands, xsize=256, ysize=256, masked=masked, affine=affine)
 
+    def get_tile(self, x_tile, y_tile, zoom,
+                 bands=None, masked=False, resampling=Resampling.cubic):
+        """Convert mercator tile to raster window.
+
+        :param x_tile: x coordinate of tile
+        :param y_tile: y coordinate of tile
+        :param zoom: zoom level
+        :param bands: list of indices of requested bands, default None which returns all bands
+        :param blocksize: tile size  (x & y) default 256, for full resolution pass None
```

Comment:
`blocksize` parameter is not presented in function signature and is not used in function itself.